### PR TITLE
common: change default file name for analysis results

### PR DESCRIPTION
**Pull request by rsarky**
Change default file name for analysis results for both mbox and patch stack mode to 'patch-groups'

### CODE REVIEW

File: common/default.cfg [hunk 1]
```diff
@@ -26,8 +26,8 @@ INVALID = "resources/invalid"
 STACK_HASHES = "resources/stack-hashes"
 PATCH_STACK_DEFINITION = "resources/patch-stack-definition.dat"
 FALSE_POSTITIVES = "resources/false-positives"
-PASTA_RESULT = "resources/pasta-result"
-MBOX_RESULT = "resources/mbox-result"
+PASTA_RESULT = "resources/patch-groups"
+MBOX_RESULT = "resources/patch-groups"
```
Comment (rralf) [inline common/default.cfg]: We can consolidate those fields to one field PATCH_GROUPS=...

**Review by rralf — CHANGES_REQUESTED** [review]

File: common/default.cfg (hunk 1 above)
Comment (rsarky) [reply common/default.cfg]: yes, makes sense

Comment (rsarky) [reply common/default.cfg]: done!

**Review by rralf — CHANGES_REQUESTED** [review]
You need to rename PreemptRT/resources/pasta-result -> PreemptrT/resources/patch-groups

**Comment by rsarky** [comment]
> You need to rename PreemptRT/resources/pasta-result -> PreemptrT/resources/patch-groups

sure!

**Comment by rralf** [comment]
Thanks, applied

**Pull request merged**